### PR TITLE
dumpling: update some desc for migrating from aurora

### PR DESCRIPTION
**Pull request by glkappe**
对 dumpling 添加必要参数，否则 Aurora 会报错
origin_img_v2_e5a124ec-55e4-4eea-87f7-9f87bac9450g

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: migrate-aurora-to-tidb.md [hunk 1]
```diff
@@ -74,7 +74,9 @@ tiup dumpling --host ${host} --port 3306 --user root --password ${password} --fi
 |-B 或 --database   |导出指定数据库|
 |-T 或 --tables-list |导出指定数据表|
 |-d 或 --no-data    |不导出数据，仅导出 schema|
-|-f 或 --filter     |导出能匹配模式的表，不可用 -T 一起使用，语法可参考[table filter](/table-filter.md)|
+|-f 或 --filter     |导出能匹配模式的表，不可与 -T 一起使用，语法可参考[table filter](/table-filter.md)|
+|--consistency     |导出期间的一致性模式，Aurora 只能使用 none|
```
Comment (ran-huang) [inline migrate-aurora-to-tidb.md:78]:
```suggestion
|--consistency     |导出期间的一致性模式，Aurora 只能使用 `none`，即不加锁 dump，不保证一致性|
```

**Review by ran-huang — APPROVED** [review]
Rest LGTM

**Review by lichunzhu — APPROVED** [review]

File: migrate-aurora-to-tidb.md [hunk 2]
```diff
@@ -74,7 +74,9 @@ tiup dumpling --host ${host} --port 3306 --user root --password ${password} --fi
 |-B 或 --database   |导出指定数据库|
 |-T 或 --tables-list |导出指定数据表|
 |-d 或 --no-data    |不导出数据，仅导出 schema|
-|-f 或 --filter     |导出能匹配模式的表，不可用 -T 一起使用，语法可参考[table filter](/table-filter.md)|
+|-f 或 --filter     |导出能匹配模式的表，不可与 -T 一起使用，语法可参考[table filter](/table-filter.md)|
+|--consistency     |导出期间的一致性模式，Aurora 只能使用 `none`，即不加锁 dump，不保证一致性|
+
```
Comment (ran-huang) [inline migrate-aurora-to-tidb.md:79]:
```suggestion
```

**Pull request merged**